consensus/bor/bor.go

```diff
@@ -1174,6 +1179,16 @@ func (c *Bor) CommitStates(
 			"txHash", eventRecord.TxHash,
 		)
 
+		go func() {
+			stateData := types.StateData{
```

Comment:
Create an object outside of the go routine.

Reply:
Just curious, why does it matter?